xonsh/completers/commands.py

```diff
@@ -7,6 +7,7 @@
 from xonsh.completers.tools import get_filter_function
 
 SKIP_TOKENS = {'sudo', 'time', 'timeit', 'which', 'showcmd', 'man'}
+GOTO_TOKENS = {'|', '||', '&&', 'and', 'or'}
```

Comment:
I don't know that "GOTO" fits my brain here, because we aren't referring to jump statements.  Maybe this it nitpicking too much but perhaps we could call them  `end_proc_tokens` or something?